docs/hugo/content/contributing/extension-points/_index.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Extension Points
+linktitle: Extension Points
+weight: 100
```

Comment:
I thought lower weight is "higher" on the page. I guess this is the index so this weight is about where this lands compared to othher contributing docs?

So this is at the end?

Just confirming this is correct as it seemed a little surprising to me